t/api/02-assets.t

```diff
@@ -137,7 +137,6 @@ la;
 
 # test delete operation
 $t->delete_ok('/api/v1/assets/' . $listing->[0]->{id})->status_is(200);
-is($t->tx->res->json->{count}, 1, "one asset deleted");
```

Comment:
This is actually a breaking API change. Do we care? Maybe you can let it still return the number of deleted assets for the sake of compatibility (and possibly consistency with other API routes)?

But I agree of course that the `count` makes no sense if we only support removing one asset at a time and return an error if that didn't work.

Reply:
I've just read the code again and noticed that we actually support deleting multiple assets at the same time. So the count should stay in my opinion.

Reply:
I remained the count. But I did not find out where did the multiple deletion and how.  The parameter is id or type and name, how to pass more than one id? using rest api such as 'api/v1/asset/7,8'? If the parameters include more than one id,  what is the form of the search condition?

Reply:
> But I did not find out where did the multiple deletion and how.

I suppose you're right after all. There's a unique constraint for the type and name (`__PACKAGE__->add_unique_constraint([qw(type name)]);`) so there should never be more than one row. But for the sake of compatibility and consistency with other routes it is good to keep the count anyways.